DomainContent/FaceToFace/Cats/meow.js

```diff
@@ -0,0 +1,68 @@
+// A script for making a kitten meow when you stand near it
+// I promise this won't get annoying
+// One of those statements is a lie
+//
+// Copyright 2018 High Fidelity Inc.
+// Author: Liv Erickson, 8/23/2018
+// 
+// Licensed under the Apache 2.0 license
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+
+(function() {
+
+    var selfEntityID;
+    var selfPosition;
+
+    var MEOW_SOUND_URL = Script.resolvePath('./resources/sounds/meow.mp3');
+
+    var MEOW_ANIMATION_URL = 'atp:/kitten_Meow.fbx';
+    var IDLE_ANIMATION_URL = 'atp:/kitten_Idle.fbx';
+
+    var meowSound = SoundCache.getSound(MEOW_SOUND_URL);
+    var meowInterval;
+
+    var MeowEngine = function(){
+
+    };
+
+    MeowEngine.prototype = {
+        remotelyCallable : ['meow'],
+
+        preload : function(entityID) {
+            selfEntityID = entityID;
+            selfPosition = Entities.getEntityProperties(entityID, 'position').position;
```

Comment:
not related, but wondering if we need to update the documentation to reflect that the second parameter can also be a single string.  Think we have it listed as only an array of strings.